pages/about.html

```diff
@@ -7,43 +7,42 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>&hearts;shell-bee&hearts;</title>
 </head>
+
 <body>
   <header>
     <h1>&hearts;about&hearts;</h1>
-    <nav>
-      <div class="dropdown">
-          <span>navigation:</span>
-          <div class="dropdown-content">
-              <a href="index.html" >home</a>
-              <br><a href="about.html">about</a>
-              <br><a href="portfolio.html">portfolio</a>
-            </div>
-        </div>
+      <nav>
+        <ul>
+          <li><a href="index.html" >home</a></li>
+          <li><a href="about.html">about</a></li>
+          <li><a href="portfolio.html">portfolio</a></li>
+        </ul>
```

Comment:
Since the `nav` and `h1` are both direct children of the `header`, we'd want them on the same level of indentation.